src/testdir/test_buffer.vim

```diff
@@ -270,6 +270,20 @@ func Test_goto_buf_with_confirm()
   close!
 endfunc
 
+func Test_confirm_write_unamed()
+  CheckUnix
+  CheckNotGui
+  call writefile(['foo'], 'Xfoo', 'D')
+  enew
+  call setline(1, 'test')
+  call feedkeys('y', 'L')
+  confirm edit Xfoo
+  call assert_equal('Xfoo', @%)
+  bprevious
+  call assert_equal('Untitled', @%)
+  close!
+endfunc
```

Comment:
Does this test actually work? For me it fails with:
```
Vim(edit):E338: Sorry, no file browser in console mode @ command line..script /home/chrisbra/code/vim-upstream/src/testdir/runtest.vim[607]..function RunTheTest[57]..Test_confirm_write_unamed, line 7
```

Reply:
also doesn't work so i wonder how i got the browser notify .. there has `CheckNotGUi` ..

Reply:
Setting fname has been simplified. Adjusted previous test cases. But still enter the browse macro branch